code/modules/hydroponics/spreading/spreading_growth.dm

```diff
@@ -90,22 +90,27 @@
 		if(prob(chance))
 			sampled = 0
 
-	if(is_mature() && !has_buckled_mobs())
-		for(var/turf/neighbor in neighbors)
-			for(var/mob/living/M in neighbor)
-				if(seed.get_trait(TRAIT_SPREAD) >= 2 && (M.lying || prob(round(seed.get_trait(TRAIT_POTENCY)))))
-					entangle(M)
-
-	if(is_mature() && neighbors.len && prob(spread_chance))
-		//spread to 1-3 adjacent turfs depending on yield trait.
-		var/max_spread = between(1, round(seed.get_trait(TRAIT_YIELD)*3/14), 3)
-
-		for(var/i in 1 to max_spread)
-			if(prob(spread_chance))
-				sleep(rand(3,5))
-				if(!neighbors.len)
-					break
-				spread_to(pick(neighbors))
+	if(is_mature())
+		if(!has_buckled_mobs())
+			for(var/turf/neighbor in neighbors)
+				for(var/mob/living/M in neighbor)
+					if(seed.get_trait(TRAIT_SPREAD) >= 2 && (M.lying || prob(round(seed.get_trait(TRAIT_POTENCY)))))
+						entangle(M)
+
+		if(seed.get_trait(TRAIT_SPORING) && prob(1))
+			visible_message(SPAN_WARNING("\The [src] hisses, releasing a cloud of spores!"), SPAN_WARNING("Something nearby hisses loudly!"))
+			seed.create_spores(get_turf(src))
+
+		if(neighbors.len && prob(spread_chance))
+			//spread to 1-3 adjacent turfs depending on yield trait.
+			var/max_spread = between(1, round(seed.get_trait(TRAIT_YIELD)*3/14), 3)
+
+			for(var/i in 1 to max_spread)
+				if(prob(spread_chance))
+					sleep(rand(3,5))
+					if(!neighbors.len)
+						break
+					spread_to(pick(neighbors))
```

Comment:
Plants already check for other plants in the turf they end up in after moving, and attempt to overgrow each-other, one of them getting deleted.

Reply:
This will only occur in areas where the vine falls/is moved by some other method (stairs), or there are vines of another species. As it is now, the latter already occurs.